internal/bminventory/inventory_test.go

```diff
@@ -761,6 +761,28 @@ var _ = Describe("IgnitionParameters", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(report.IsFatal()).To(BeFalse())
 		})
+
+		It("Doesn't include the static ip data for minimal isos", func() {
+			cluster.ImageInfo.StaticIpsConfig = "mac1;ip1;dns1;gw1;mask1\nmac2;ip2;dns2;gw2;mask2"
```

Comment:
What about IPv6? This will work thanks to backward compatibility, but IMO the full format must be tested. Also, notice the mask's position. Possible variants:
IPv4-only partial `mac;ipV4;maskV4;dnsV4;gwV4`
IPv4-only full `mac;ipV4;maskV4;dnsV4;gwV4;;;;`
IPv4+IPv6 `mac;ipV4;maskV4;dnsV4;gwV4;ipV6;maskV6;dnsV6;gwV6`
IPv6-only `mac;;;;;ipV6;maskV6;dnsV6;gwV6`

Reply:
It is entirely irrelevant for this test, but yes I can fix up the format here.